.travis.yml

```diff
@@ -15,13 +15,8 @@ before_script:
   - cd ..
 
 script:
-  - make clean
-  - make
-  - cd src
-  - mkdir -p ../build
-  - module_packer -v -o "../build/redisgraph.{os}-{architecture}.latest.zip" "`pwd`/redisgraph.so"
-  - cd ..
-  # - make test
+  - make test
```

Comment:
What about simply calling `make` to build the module?

Reply:
Also I don't think test target is defined within our makefile(s)

Reply:
1. Right, I forgot to define it.
2. I just made it the same way it was done in redisearch. They call make in before_script, and then make test and make package in script

Reply:
make in the before script builds redis,
I don't think our module is getting built.